packages/components/src/PopupTrigger.tsx

```diff
@@ -26,6 +28,10 @@ type DivProps = JSX.IntrinsicElements['div'];
 // aria-popup supports a limited subset of ARIA roles.
 type PopupRole = Exclude<AriaAttributes['aria-haspopup'], boolean | 'true' | 'false'>;
 
+export const makeMenuIcon = (iconName: IconId, props?: IconProps | undefined) => {
+  return icon(iconName, { ...{ size: 15, style: { marginRight: '.3rem' } }, ...props });
```

Comment:
Do we want a test with this?

If so, I am unsure what to test.

If I test that I can find the resulting icon component, it feels like I am more testing the icon function.

Am I testing that icon got called with the right parameters? As in, did my replacement of Lodash's merge work correctly?  I know that I can do that in Junit, not sure if there's a straightforward way of doing this in Jest, and if so, is it good to have that test